lib/maze/server.rb

```diff
@@ -173,6 +182,7 @@ def start
             server.mount '/builds', Servlets::Servlet, :builds
             server.mount '/uploads', Servlets::Servlet, :uploads
             server.mount '/sourcemap', Servlets::Servlet, :sourcemaps
+            server.mount '/trace', Servlets::Servlet, :traces
```

Comment:
Wouldn't `/traces` be more fitting? (following `/sessions` and `/logs`)

Reply:
Changed to `/traces` - makes sense when `/sessions` and `/logs` are used more than `/sourcemap`.